### PR TITLE
feat(css-formatter): support function

### PR DESCRIPTION
**Pull request by yeonjuan**
## Summary

It's a pr supporting for CSS function formating.

## Test Plan

test case.

### CODE REVIEW

File: internal/css-parser/index.ts [hunk 1]
```diff
@@ -914,6 +914,17 @@ function parseComponentValue(
 		);
 	}
 
+	if (matchToken(parser, "Comma")) {
```
Comment (ematipico) [inline internal/css-parser/index.ts:917]: I think the Comma should have its own node: https://www.w3.org/TR/css-syntax-3/#parse-comma-separated-list-of-component-values

File: internal/formatter/test-fixtures/css/function/input.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (ematipico) [inline internal/formatter/test-fixtures/css/function/input.css:1]: I think we should add also a nested function case:

```css
padding: calc(calc(8px-2px) + 8px);
```

**Review by ematipico — CHANGES_REQUESTED** [review]
Thank you! Just a couple of suggestions

**Comment by yeonjuan** [comment]
@ematipico Thanks for the review. all fixed :)

**Review by ematipico — APPROVED** [review]
Thank you!!

**Pull request merged**